src/main/java/com/frcteam3636/frc2024/subsystems/climber/ClimberIO.kt

```diff
@@ -26,9 +29,42 @@ interface ClimberIO {
     fun moveClimber(speed: Double)
 }
 
+class ClimberIOSim : ClimberIO {
+    companion object {
+        //TODO: Find all of these
+        const val ELEVATOR_KV = 0.1
+        const val ELEVATOR_KA = 0.1
+        const val ELEVATOR_MAX_HEIGHT = 1.0
+        const val ELEVATOR_MIN_HEIGHT = 0.0
+        const val ELEVATOR_START_HEIGHT = 0.5
+    }
+
+    private var climberMotor = DCMotor.getNEO(1)
+    private var elevatorSim = ElevatorSim(
+        ELEVATOR_KV,
+        ELEVATOR_KA,
+        climberMotor,
+        ELEVATOR_MIN_HEIGHT,
+        ELEVATOR_MAX_HEIGHT,
+        true,
+        ELEVATOR_START_HEIGHT
+    )
+
+    override fun updateInputs(inputs: ClimberIO.ClimberInputs) {
+        inputs.climberPosition = elevatorSim.positionMeters
+    }
+
+    override fun moveClimber(speed: Double) {
+        elevatorSim.setInputVoltage(12.0 * speed)
+        elevatorSim.update(0.02)
+    }
+}
+
 class ClimberIOReal : ClimberIO {
     companion object {
         const val CLIMBER_GEAR_RATIO = 1.0
+        //TODO: Find this value
+        const val METERS_PER_ROTATION = 0.003175;
```

Comment:
It's not great to name variables by _only_ their unit. I suggest `ELEVATOR_TRAVEL_PER_ENCODER_ROTATION`.